### PR TITLE
Kyles

### PR DESCRIPTION
**Pull request by kegillett**

### CODE REVIEW

File: answers/javaScript.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ## JavaScript
 * `null` vs `undefined` - Null means there is 'no value.' Undefined represents a variable that has not been declared or not given a value.
 	* What does `type of null` output? It returns 'object.'
+* `==` vs `===` - == compares values but not data types. eg. 3 == "3" would be true. ie... it converts the values.
```
Comment (ed42311) [inline answers/javaScript.md]: This feels like it needs some line breaks and more styling to make it more readable.

File: answers/javaScript.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ## JavaScript
 * `null` vs `undefined` - Null means there is 'no value.' Undefined represents a variable that has not been declared or not given a value.
 	* What does `type of null` output? It returns 'object.'
-* `==` vs `===` - == compares values but not data types. eg. 3 == "3" would be true. ie... it converts the values.
-=== compares values including data types. 3 === "3" would be false ie.... it doesn't convert values.
+* `==` vs `===`
+		- == compares values but not data types. eg. 3 == "3" would be true. ie... it converts the values.
+		-	=== compares values including data types. 3 === "3" would be false ie.... it doesn't convert values.
```
Comment (jack829) [inline answers/javaScript.md]: @kegillett put anything that represents code inside "`" (backticks)

Comment (jack829) [reply answers/javaScript.md]: so `===` or `3 === "3"`

Comment (kegillett) [reply answers/javaScript.md]: fixed, noted and will push again, thanks Jack.

**Comment by kegillett** [comment]
i guess im confused on what need to happen now with this branch..

**Pull request closed without merging**